PhotoMap/Features/Map/ViewModels/MapViewModel.swift

```diff
@@ -36,6 +41,18 @@ class MapViewModel: MapViewModelType {
         self.diContainer = diContainer
         self.locationService = diContainer.resolve()
 
+        self.firestoreService = diContainer.resolve()
+        self.сompletionHandler = { completion in
+            switch completion {
+            case .failure(let error):
+                // coordinator.errorAlertSubject.send(error)
+                print(error.localizedDescription)
+            case .finished:
+                return
+            }
+        }
```

Comment:
Not sure that's a good idea of using shared completion handler. Just use closures inside methods where it's called or extract code to another method